ppl/archive/multisource.go

```diff
@@ -128,9 +128,10 @@ func (m *spanMultiSource) SourceFromRequest(ctx context.Context, req *api.Worker
 			return nil, zqe.E(zqe.Invalid, "invalid chunk path: %v", p)
 		}
 		uri := tsdir.path(m.ark)
-		md, err := chunk.ReadMetadata(ctx, chunk.MetadataPath(uri, id), m.ark.DataOrder)
+		mdPath := chunk.MetadataPath(uri, id)
+		md, err := chunk.ReadMetadata(ctx, mdPath, m.ark.DataOrder)
 		if err != nil {
-			return nil, err
+			return nil, zqe.E("failed to read chunk metadata: %s %w", mdPath.String(), err)
```

Comment:
An error got thrown here a couple of times during test development, so we added a better message so we can track it down if it happens again. I would like to be able to reproduce it.